lib/python/rose/config_processors/jinja2.py

```diff
@@ -63,6 +77,12 @@ def process(self, conf_tree, item, orig_keys=None, orig_value=None, **_):
                 except UnboundEnvironmentVariableError as exc:
                     raise ConfigProcessError([s_key, key], node.value, exc)
                 tmp_file.write(self.ASSIGN_TEMPL % (key, value))
+            environ = kwargs.get("environ")
+            if environ:
+                tmp_file.write('[cylc]\n')
+                tmp_file.write('    [[environment]]\n')
```

Comment:
Why not `[runtime][root][environment]`

Reply:
The environment variables from `rose-suite.conf` are currently implemented as environment variables for the suite server program, not for tasks. So `[cylc][environment]` offers the best backward compatibility (and hence less chance of breakage). We can decide to do `[runtime][root][environment]` as well in the future, but I am not sure I want to risk it at this stage.